### PR TITLE
Inlined default countries data.

### PR DESCRIPTION
**Pull request by SaadHassan-dev**

### CODE REVIEW

**Comment by UmanShahzad** [comment]
@SaadHassan-dev lemme know if we can get unit tests fixed, would like to merge soon

**Comment by SaadHassan-dev** [comment]
> @SaadHassan-dev lemme know if we can get unit tests fixed, would like to merge soon

Done.

File: lib/ipinfo/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module IPinfo
-    VERSION = '1.1.0'
+    VERSION = '1.1.1'
```
Comment (UmanShahzad) [inline lib/ipinfo/version.rb:4]: We'll need a major version bump now as we have a backwards incompatible change - the meaning of the settings we updated is now to expect an in-memory map and not a file.

**Review by UmanShahzad — APPROVED** [review]

File: lib/ipinfo/version.rb (hunk 1 above)
Comment (SaadHassan-dev) [reply lib/ipinfo/version.rb]: Done.

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**